docs/develop/topics/theming.md

```diff
@@ -0,0 +1,58 @@
+# Theming 
+
+This page will provide you with in-depth information about the Invenio RDM theme structure, which is inheriting the concepts from the Semantic UI
+
+## Semantic UI
+
+### Concepts
+
+From semantic UI docs:
+
+> Semantic UI treats words and classes as exchangeable concepts.
+Classes use syntax from natural languages like noun/modifier relationships, word order, and plurality to link concepts intuitively.
+
+It basically refers to using the CSS classes to describe how the element looks like, not what it is, f.e.
+
+```html
+<div class="ui label right floated"></div>
+```
+
+tells us more about what to expect from the final view, rather than
+
+```html
+<div class="metadata-label"></div>
+```
+
+which has significant influence on the maintainability of the styling - and it is very important in big projects. It is easier for any developer to remember a few "behavioral" classes, rather than to remember that metadata-label will be always right floated.
```

Comment:
```suggestion
which has significant influence on the maintainability of the styling - and it is very important in large projects. It is easier for any developer to remember a few "behavioral" classes, rather than to remember that `metadata-label` will be always right floated.
```